### PR TITLE
XHTML5 support for maven-doxia

### PR DESCRIPTION
**Pull request by minfrin**
https://issues.apache.org/jira/browse/DOXIA-575

### CODE REVIEW

File: doxia-core/src/main/java/org/apache/maven/doxia/markup/HtmlMarkup.java [hunk 1]
```diff
@@ -25,11 +25,13 @@
 /**
  * List of <code>Html</code> tags.
  * <p>
- *   This should contain all valid XHTML 1.0 tags, comprising the tags in
- *   {@link javax.swing.text.html.HTML.Tag} plus several others.
+ * This should contain all valid XHTML 1.0 and HTML5 tags, comprising the tags
```
Comment (michael-o) [inline doxia-core/src/main/java/org/apache/maven/doxia/markup/HtmlMarkup.java]: Does it really contain XHTML 1.0?

File: doxia-core/src/main/java/org/apache/maven/doxia/parser/XhtmlBaseParser.java [hunk 2]
```diff
@@ -77,9 +77,6 @@
     /** Used to wrap the definedTerm with its definition, even when one is omitted */
     boolean hasDefinitionListItem = false;
 
-    /** Decoration properties, eg for texts. */
-    private final SinkEventAttributeSet decoration = new SinkEventAttributeSet();
```
Comment (michael-o) [inline doxia-core/src/main/java/org/apache/maven/doxia/parser/XhtmlBaseParser.java]: Can you please explain why this is gone and most has been turned into `sink#inline()`?

File: doxia-core/src/test/java/org/apache/maven/doxia/parser/XhtmlBaseParserTest.java [hunk 3]
```diff
@@ -555,42 +520,72 @@ public void testLists()
     }
 
     /** @throws Exception  */
-    public void testStyles()
+    public void testSimpleTags()
         throws Exception
```
Comment (michael-o) [inline doxia-core/src/test/java/org/apache/maven/doxia/parser/XhtmlBaseParserTest.java]: Why is this test gone?

**Comment by minfrin** [comment]
> * Why have specialized items replaced with inline? What is the purpose?

They were all almost identical variations of what would otherwise have been the "span" tag. As I recall (this patch was written 10 months ago) it extended an existing pattern in the code.

> * Several `@since` tag value are wrong
> * There is a merge conflict, please resolve

Unfortunately this PR bit-rotted, the most recent patch is at 
https://jira.apache.org/jira/secure/attachment/12945908/DOXIA-575.patch

**Comment by michael-o** [comment]
Please perform the requested changes by applying your patch and I will re-review.

**Comment by minfrin** [comment]
This PR has been replaced with https://github.com/apache/maven-doxia/pull/16.

**Pull request closed without merging**